content/en/docs/pipelines/installation/overview.md

```diff
@@ -116,10 +117,17 @@ Kubeflow Pipelines updates.
 | 0.7.0                  | 0.1.31                     |                                         |
 | 1.0.0                  | 0.2.0                      |                                         |
 | 1.0.2                  | 0.2.5                      |                                         |
-| 1.1.0 (To be released) | 1.0.0*                     | Some platforms may still have Kubeflow Pipelines 0.2.5 |
+| 1.1.0                  | 1.0.0                      | Some platforms may still have Kubeflow Pipelines 0.2.5 |
```

Comment:
Would it make sense to have specific columns about the different KF distributions (or some subset) so you could be exact about which pipeline version is installed. One way to do this would be to have columns linking to the various deploy instructions

e.g. one column for GCP, one for AWS, one for openshift etc...

You would only include columns for the distributions you wanted to be specific about.

Arguably if a distribution isn't installing KFP 1.0.0 then its not KF 1.1.0

Reply:
It is hard to keep this information up-to-date in KFP documentation (also it's hard for me too, because I'm not familiar with status of other platforms), so I just want to mention the warning here.

> Arguably if a distribution isn't installing KFP 1.0.0 then its not KF 1.1.0

I agree with this. Let me first mention platforms that supported KFP 1.0.0